spec/datadog/ci/ext/environment_spec.rb

```diff
@@ -4,18 +4,58 @@
 require 'datadog/ci/ext/environment'
 
 RSpec.describe Datadog::CI::Ext::Environment do
+  fixture_dir = "#{File.dirname(__FILE__)}/fixtures/"
+
   describe '.tags' do
-    subject(:tags) { described_class.tags(env) }
+    subject(:tags) do
+      ClimateControl.modify(environment_variables) { described_class.tags(env) }
+    end
 
-    Dir.glob("#{File.dirname(__FILE__)}/fixtures/ci/*.json") do |filename|
+    let(:env) { {} }
+    let(:environment_variables) { {} }
+
+    shared_context 'without git installed' do
+      before { allow(described_class).to receive(:`).and_raise(Errno::ENOENT, 'git') }
+    end
+
+    Dir.glob("#{fixture_dir}/ci/*.json") do |filename|
```

Comment:
I've been thinking that it may not be a bad idea to sort these files before we run the test, as otherwise we will have an uncontrolled source of randomness, which may lead to some confusion if one is trying to reproduce a failure (hopefully we'll never need it, but since a call to sort is so cheap here...).

Reply:
This makes sense.